test/ejabberd_config_SUITE.erl

```diff
@@ -25,11 +26,16 @@ groups() ->
                          add_a_module,
                          delete_a_module,
                          reload_a_module]},
+     {reload_cluster, [], [cluster_smoke,
+                           change_module_option_with_node_param_opts,
+                           change_module_option_with_node_specific_mods,
+                           module_deps_work_correctly_with_reload_cluster]},
      {odbc_pools, [], [odbc_server_no_pools,
                           odbc_server_pools]}
     ].
 
 init_per_suite(Config) ->
+    stringprep:start(),
```

Comment:
`application:ensure_all_started`?

Reply:
There is no one way to do it:

```bash
git grep stringprep:start                                                           
test/acl_SUITE.erl:    ok = stringprep:start(),                  
test/auth_external_SUITE.erl:    ok = stringprep:start(),        
test/auth_http_SUITE.erl:    ok = stringprep:start(),            
test/auth_tokens_SUITE.erl:    ok = stringprep:start(),          
test/component_reg_SUITE.erl:    ok = stringprep:start(),        
test/ejabberd_c2s_SUITE.erl:    ok = stringprep:start(),         
test/ejabberd_config_SUITE.erl:    stringprep:start(),           
test/ejabberd_sm_SUITE.erl:    ok = stringprep:start(),          
test/jlib_SUITE.erl:    ok = stringprep:start(),                 
test/keystore_SUITE.erl:    ok = stringprep:start(),             
test/mod_aws_sns_SUITE.erl:    stringprep:start(),               
test/mongoose_config_SUITE.erl:    stringprep:start(),           
test/privacy_SUITE.erl:    ok = stringprep:start(),              
test/roster_SUITE.erl:    ok = stringprep:start(),               
test/router_SUITE.erl:    ok = stringprep:start(), 
```

Reply:
That's why I'm suggesting, not "requesting change". `:start` is good enough. :)